pkg/apis/extensions/types.go

```diff
@@ -525,6 +525,29 @@ type JobList struct {
 	Items []Job `json:"items"`
 }
 
+// JobTemplate describes a template for creating copies of a predefined pod.
+type JobTemplate struct {
```

Comment:
That will be done in a separate PR, this PR was initiated by @sdminonne request and the move was mentioned just recently.